release: qaseio 2.1.3 , qase-javascript-commons 2.0.9 and qase-playwright 2.0.3 (#615)
qase-tms/qase-javascript: qase-javascript-commons/changelog.md

# qase-javascript-commons@2.0.9

## What's new

> Added new configuration options `rootSuite` for specify root suite.
> This option is available in config file and `QASE_ROOT_SUITE` env variable

Reviewer: ```suggestion
Added new configuration option `rootSuite` to specify a root suite.
This option is available in the config file and the `QASE_ROOT_SUITE` env variable
```

Author: Fixed

> This option is available in the config file and the `QASE_ROOT_SUITE` env variable

Reviewer: Does the root suite apply to local reports? If not, it should probably belong to the testops config sections and be named `QASE_TESTOPS_ROOT_SUITE`.

Author: yes, the root suite applies to both reporters: testops and local

```diff
{
  "mode": "testops",
  "fallback": "report",
  "environment": "local",

> + "rootSuite": "Root Suite",

Reviewer: Can I use a path with nested suites?

Author: no, only one suite
